src/ngx_http_lua_socket_tcp.c

```diff
@@ -4908,65 +4901,30 @@ ngx_http_lua_socket_push_input_data(ngx_http_request_t *r,
     nbufs = 0;
     ll = NULL;
 
+    luaL_buffinit(L, &luabuf);
     for (cl = u->bufs_in; cl; cl = cl->next) {
+        ptrdiff_t chunk_size;
```

Comment:
> @lneto I mean we need to be consistent with nginx's code convention wherever possible. Being creative and distinguished here does more harm than good, no matter how much you like it personally.

There is nothing of creativity in using `ptrdiff_t` for pointers difference or on specifying the scope of a variable. Also, I don't see how you have concluded that it is a convention.

>@lneto And BTW, I don't see why these two points cannot be part of "coding style".

Simply because coding style is about syntax, not semantics.